src/uu/cp/Cargo.toml

```diff
@@ -28,6 +28,15 @@ uucore = { workspace = true, features = ["entries", "fs", "perms", "mode"] }
 walkdir = { workspace = true }
 indicatif = { workspace = true }
 
+[target.'cfg(target_os = "windows")'.dependencies]
+windows-sys = { workspace = true, features = [
```

Comment:
I'll admit I know little about these crates, but could we use `windows` crate instead of `windows-sys`?

Reply:
I used -sys because looking through the other crates in uuitils/coreutils I saw they used -sys (note that it looks like there are two versions of windows-sys in the dependency tree, I bet a cargo update might take care of that, but I didn't want to mess with the lockfile.

In theory windows-sys is supposed to compile faster than windows, but especially for the Win32 API the differences are not that large, and sometimes the "extra help" actually makes the code longer (you need to create more wrapper types, and integration between "rust" types and "windows" types is not always great).

I actually did write a version using normal windows and it was basically the same amount of code. The only really useful feature was the windows `BOOL` type has a `.ok()` method that makes a result containing the error from GetLastError() if the `BOOL` is zero. I see no reason to use windows instead of windows-sys just for that.

Reply:
> I used -sys because looking through the other crates in uuitils/coreutils I saw they used -sys

This is a bit unfortunate. I think we should push for more high-level APIs in general.

> note that it looks like there are two versions of windows-sys in the dependency tree

`cargo update` doesn't fix that, but we keep track of it via `cargo deny`, so we are aware of this.

As for the benefits, it might not be shorter, but it might fit in better with the rest of the code by being more "rusty". Does the `windows` crate allow us to remove any `unsafe`?

Reply:
Just to note: I don't think this is a blocker for this PR.

Reply:
THe windows crate does not allow us to remove any usage of unsafe

Reply:
taplo is complaining about:

ERROR taplo:format_files: the file is not properly formatted path="/home/runner/work/coreutils/coreutils/src/uu/cp/Cargo.toml"